django-verdant/rca/templatetags/rca_navigation_tags.py

```diff
@@ -0,0 +1,89 @@
+"""Provides the {% navigation_via_api %} template tag.
+
+The new rebuild site is where the menu is being managed and updated.
+Instead of copying and pasting rendered html when the new site's menu gets
+updated, this template tag will request navigation data from the new site's
+navigation API endpoint. Instead of making this API call on every page request
+it's stored in cache['navigation_from_api'].
+
+"""
+
+import logging
+import requests
+from requests.auth import HTTPBasicAuth
+
+from django.conf import settings
+from django import template
+from django.core.cache import cache
+
+
+register = template.Library()
+
+logger = logging.getLogger("verdant-rca")
+
+
+class CantPullFromRcaApi(Exception):
+    pass
+
+
+def parse_navigation(data):
+    # Temporarily in place should we need to do any further parsing of the data
+    return data
```

Comment:
Depending on the FE rendering I might need to do some extra parsing, if not this will be removed